challenges/unit-testing/writing-tests/remove-vowels/remove-vowels-in-array.js

```diff
@@ -1,7 +1,7 @@
-let removeVowels = require("./remove-vowels");
-
 function removeVowelsFromWords(words) {
-  return removeVowels(words);
+  return words.map(function (word) {
+    return word.replace(/[aeiou]/gi, "");
```

Comment:
Ok this also works. FYI what I would have done: keep this line "word.replace(..)" in remove-vowels.js, so that it removes vowels from a single word, no array. And here I can call words.map(removeVowels).
The point is having the function in this file take an array, and returning an array. And having the function in remove-vowels.js take a string, and return a string.
But again you code is good enough